.gitignore

```diff
@@ -27,6 +27,7 @@ theme_compiled.css
 *_admin_thumbnail.gif
 *_admin_thumbnail.jpg
 *_admin_thumbnail.png
+fabfile_personal.py
```

Comment:
Not important, but could we call it `local_fabfile.py` to be in line with `local_settings.py`?